database/migrations/2022_08_25_213308_adds_ldap_default_group_to_settings_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddsLdapDefaultGroupToSettingsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('settings', function (Blueprint $table) {
+            $table->integer('ldap_default_group')
+                ->after('ldap_basedn');
```

Comment:
We probably also want to make this nullable, with a default of null. Different versions of mysql can get cranky if we don't explicitly say that.

Reply:
defaulted to null!